src/workerd/io/compatibility-date.capnp

```diff
@@ -703,4 +703,13 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("nonclass_entrypoint_reuses_ctx_across_invocations")
       $compatEnableDate("2025-03-10");
   # Creates a unique ExportedHandler for each call to `export default` thus allowing a unique ctx per invocation
+
+  queueConsumerNoWaitForWaitUntil @75 :Bool
+      $compatEnableFlag("queue_consumer_no_wait_for_wait_until")
+      $compatDisableFlag("queue_consumer_wait_for_wait_until");
```

Comment:
Is it intentional that this does not have a default on date?

Reply:
It was intentional since PM had wanted to see how this worked for some specific users before making it the default, but I'll double check.

Assuming that hasn't changed, I've left a comment to that effect for now.